libp2p/Common.cpp

```diff
@@ -19,6 +19,7 @@
  * @date 2014
  */
 
+#include <regex>
```

Comment:
Please apply clang-format to your changes, it should put this below other includes

Reply:
Thanks for the tip, I tend to not do that since it causes a lot of churn in some files (e.g. libethereum\Blockchain.cpp) but I'll do so in this case.

Reply:
> Thanks for the tip, I tend to not do that since it causes a lot of churn in some files (e.g. libethereum\Blockchain.cpp) but I'll do so in this case.

Actually, it causes a lot of churn in this file as well...I'll manually move the #include.

Reply:
> You should appply it only to your changes not

Thanks, I'll take another look at that.